giraffa-core/src/main/java/org/apache/giraffa/hbase/ClientNamenodeProtocolServerSideCallbackTranslatorPB.java

```diff
@@ -236,19 +280,7 @@ public void addBlock(RpcController controller, AddBlockRequestProto req,
       done.run(blockingTranslator.addBlock(controller, req));
     } catch (ServiceException e) {
       handleRemoteException(controller, e, done,
-          AddBlockResponseProto.newBuilder().setBlock(
-              LocatedBlockProto.newBuilder()
-              .setB(ExtendedBlockProto.newBuilder()
-                  .setPoolId("")
-                  .setBlockId(0)
-                  .setGenerationStamp(0))
-              .setOffset(0)
-              .setCorrupt(false)
-              .setBlockToken(TokenProto.newBuilder()
-                  .setIdentifier(ByteString.EMPTY)
-                  .setPassword(ByteString.EMPTY)
-                  .setKind("")
-                  .setService(""))).build());
+          GiraffaPBHelper.addBlock());
```

Comment:
Can you tell, why we need to send fake object there instead of null. Looks a bit strange to me.